src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -231,13 +231,29 @@ class JesBackend extends Backend with LazyLogging {
     }
   }
 
-  private def runWithJes(backendCall: BackendCall, command: String, jesInputs: Seq[JesInput], jesOutputs: Seq[JesOutput]): ExecutionResult = {
-    val tag = makeTag(backendCall)
-    val jesParameters = standardParameters(backendCall.callGcsPath) ++ gcsAuthParameter(backendCall.workflowDescriptor) ++ jesInputs ++ jesOutputs
-    logger.info(s"$tag `$command`")
-    JesConnection.storage.uploadObject(backendCall.gcsExecPath, command)
+  @annotation.tailrec
+  private def createJesRun(backendCall: BackendCall, jesParameters: Seq[JesParameter], retries: Int, delaySeconds: Int): Try[Run] = {
+    Try {
+      val pipeline = Pipeline(
+        s"/bin/bash exec.sh > $LocalStdoutValue 2> $LocalStderrValue",
+        backendCall.workflowDescriptor,
+        backendCall.key,
+        jesParameters,
+        googleProject(backendCall.workflowDescriptor),
+        JesConnection
+      )
+      pipeline.run
+    } match {
+      case x: Success[Run] => x
+      case f: Failure[_] if retries > 1 =>
+        logger.warn(s"${makeTag(backendCall)} Exception occurred while creating JES Run.  Retrying in $delaySeconds seconds...", f.exception)
+        Thread.sleep(delaySeconds * 1000)
```

Comment:
That reminds me (he says, veering off course of being directly applicable here) - normally w/ exponential backoff you want to use some sort of jitter. I figured it wasn't a big deal in these cases because that jitter is more to avoid the problem where 2 things might be backing off at the same rate but here it seems likely that we're the only party backing off. Does that sound right?